src/test/ui/unsized-locals/unsized-locals-using-unsized-fn-params.rs

```diff
@@ -0,0 +1,7 @@
+#![feature(unsized_fn_params)]
+
```

Comment:
```suggestion

// Ensure that even with unsized_fn_params, unsized locals are not accepted.

```
Though I peronally would have merged this test and unsized-local-pat into one -- the common theme is "make sure `unsized_fn_params` does not accept too many things".